framework/external/easylogging/easylogging++.h

```diff
@@ -725,8 +725,14 @@ enum class LoggingFlag : base::type::EnumType {
   AutoSpacing = 8192,
   /// @brief Preserves time format and does not convert it to sec, hour etc (performance tracking only)
   FixedTimeFormat = 16384,
-  // @brief Ignore SIGINT or crash
+  //  @brief Ignore SIGINT or crash
```

Comment:
What's up with the extra space?

Reply:
All other similar lines were with like this, so I aligned to the rest of the lines.